lambdas/db-migration/src/index.ts

```diff
@@ -16,30 +18,54 @@ const getRequiredEnvVar = (name: string, env: NodeJS.ProcessEnv): string => {
   throw new Error(`The ${name} environment variable must be set`);
 };
 
-const getConnectionConfig = (env: NodeJS.ProcessEnv): Knex.PgConnectionConfig => ({
-  host: getRequiredEnvVar('PG_HOST', env),
-  user: getRequiredEnvVar('PG_USER', env),
-  // TODO Get this value from secrets manager
-  password: getRequiredEnvVar('PG_PASSWORD', env),
-  database: getRequiredEnvVar('PG_DATABASE', env),
-});
+const getSecretConnectionConfig = async (SecretId: string): Promise<Knex.PgConnectionConfig> => {
+  const secretsManager = new AWS.SecretsManager();
+  const response = await secretsManager.getSecretValue(
+    { SecretId } as AWS.SecretsManager.GetSecretValueRequest
+  ).promise();
+  if (response.SecretString === undefined) {
+    throw new Error('Database credentials are undefined!');
```

Comment:
Should we say something more specific about how the secret `SecretId` for database credentials could not be parsed? It would give more helpful information for anyone trying to debug a failure, but I'm not sure if that's leaking too much information. But I think anyone running this Lambda we're okay with seeing that information

Reply:
It's not so much it cannot be parsed in this line, as it didn't return a secret.    

I think we need to explicitly document the structure of this secret.... as for where.... 🤔  -- but *do* keep in mind that for the moment AWS SM if you create a secret via the console and tell it it's a postgres secret, it builds a form and JSONifies it for you - so there's some ... precedent for this format.      

In either case, if this isn't done right, it'll either throw a JSON.parse error, or the db connection will fail (and tell you as much) further on.

Reply:
Do we at least want to say that `No database credentials could be found for ${secret}`? So the error identifies the secret it attempted tor ead?

Reply:
3192e64a3 should probably cover both the key cases and do an up-front check